.examples/yii/config/common/routes.php

```diff
@@ -7,7 +7,7 @@
 use Yiisoft\Router\Route;
 
 return [
-    Group::create('/{_language}')
+    Group::create()
```

Comment:
changed in the app skeleton example: https://github.com/yiisoft/app/commit/dac04b189868b962f3f7328a55d7f9be9af3f0cd